src/components/Search/SearchOverTime.tsx

```diff
@@ -72,12 +72,9 @@ export default function SearchOverTime({
       <Grid
         item
         xs={2}
-        className={`$new-search-dropdown ${styles.dropDown} label-edit-wrapper`}
+        className={`new-search-dropdown ${styles.dropDown} label-edit-wrapper`}
       >
-        <InputLabel
-          id='select-repository-label--time'
-          className='dropdown-select-label'
-        >
+        <InputLabel id={selectLabel} className='dropdown-select-label'>
```

Comment:
It doesn't seem correct to use the label as the id. What happens if there are spaces for example? 

Do we really need an id though? Can we just remove it?

Reply:
I'll create a unique id name aside from the select label to account for the possibility of spaces in the name. And yes, we want an` id` for the label to match the select `labelID.` Please see https://mui.com/material-ui/react-select/